src/Kmd.Logic.Gateway.Automation.Tool/CommonOptions.cs

```diff
@@ -0,0 +1,16 @@
+﻿using CommandLine;
+
+namespace Kmd.Logic.Gateway.Automation.Tool
+{
+    internal class CommonOptions
+    {
+        [Option('t', "token", Required = true, HelpText = "Token used for authentication with Logic.")]
+        public string AuthToken { get; set; }
+
+        [Option('s', "subscriptionId", Required = true, HelpText = "Subscription ID in Logic.")]
+        public string SubscriptionId { get; set; }
+
+        [Option("verbose", HelpText = "Logs everything")]
+        public bool Verbose { get; set; }
```

Comment:
provider Id is missing as parameter, you can add this later also :)

Reply:
Yeah. Here is just a skeleton.